### PR TITLE
[bitnami/flux] Skip tests if platform is air-gapped

### PR DESCRIPTION
**Pull request by jotamartos**
### Description of the change

Skip test if platform is in an air-gapped environment

### Benefits

We can verify other functionalities of the application when skipping the failing tests

### Possible drawbacks

None

### Additional information

We dropped support to some solutions in air-gapped environment due to specific issues of the environment. We want to update the tests to be able to verify the solutions there as well

### Checklist

- [NA] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [NA] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by jotamartos** [comment]
Tests succeeded [here](https://github.com/bitnami/charts/actions/runs/6416456982/job/17420266536?pr=19762)

File: .vib/flux/goss/goss-wait.yaml [hunk 1]
```diff
@@ -9,3 +10,13 @@ addr:
   tcp://podinfo:9898:
     reachable: true
     timeout: 2000
```
Comment (FraPazGal) [inline .vib/flux/goss/goss-wait.yaml:12]: Could you give a bit of context about this `podinfo`? Is this launched as part of the `Dokuwiki` chart?

**Review by FraPazGal — CHANGES_REQUESTED** [review]
Thanks for the PR @jotamartos! I have some concerns with these changes, could you please check my comments?

File: .vib/flux/goss/goss-wait.yaml (hunk 1 above)
Comment (joancafom) [inline .vib/flux/goss/goss-wait.yaml:12]: Is `podinfo` available in an `air-gapped` environments as well? If that is the case, we can leave this one for `air-gapped` and leave the other related to the chart for other environments

Comment (jotamartos) [reply .vib/flux/goss/goss-wait.yaml:12]: it's a pod deployed via a repo that is cloned and the clone doesn't work in air-gapped envs

https://github.com/stefanprodan/podinfo/tree/master/kustomize

File: .vib/flux/runtime-parameters.yaml [hunk 2]
```diff
@@ -81,7 +81,7 @@ imageReflectorController:
 extraDeploy:
   # This is for kustomize-controller and source-controller
   # Source: https://fluxcd.io/flux/components/kustomize/kustomization/
-  - apiVersion: source.toolkit.fluxcd.io/v1beta2
+  - apiVersion: source.toolkit.fluxcd.io/v1
```
Comment (jotamartos) [inline .vib/flux/runtime-parameters.yaml:84]: Got warnings when installing the chart 

```
➜  flux git:(fluxcd-source-controller-disable-goss-if-airgapped) helm install flux -f ../../.vib/flux/runtime-parameters.yaml bitnami/flux
W1010 15:09:15.760231   65062 warnings.go:70] v1beta2 GitRepository is deprecated, upgrade to v1
W1010 15:09:15.761922   65062 warnings.go:70] v1beta2 GitRepository is deprecated, upgrade to v1
W1010 15:09:15.768191   65062 warnings.go:70] v1beta2 Kustomization is deprecated, upgrade to v1
W1010 15:09:15.769711   65062 warnings.go:70] v1beta2 Receiver is deprecated, upgrade to v1
```

**Comment by jotamartos** [comment]
Verification succeeded again [here](https://github.com/bitnami/charts/actions/runs/6470167302/job/17565756761?pr=19762)

**Review by FraPazGal — APPROVED** [review]
LGTM!

**Pull request merged**